### ISSUE TITLE
[Feature request] Command line handling

### ISSUE THREAD

**Issue by Dazaisan99** [tramhao/termusic#101]
Please keep in mind that I have just started using the software.

I'm coming from [mocp](https://github.com/jonsafari/mocp) and I'm just missing one feature from what I had before, more command line handling.

The thing you could do with mocp was type
```bash
mocp -f
mocp -G
```
And mocp would skip to the next song, or pause / unpause it.

It would be nice to have something like that in termusic, from the command line, such as :
```
termusic --next / -n
termusic --toggle-pause / -T
````
I also would be useful for WM users, as it allows to map keybindings to control the music player, as I could do with mocp.

BTW : termusic is awesome so far !

**Comment by tramhao** [comment]
Thanks for your great feedback. About the function, seems it need a separate daemon running, to accept the command from another thread. It'll need a server/client structure, thus it could take some time. But I'll keep it in the todo list. 
Currently, the function you requested can be achieved through mpris. I'll list what I'm doing right now:
1. I'm running leftwm, a window manager written in rust. I use polybar, and I setup a widget with mpris player.
2. In this widget, I can click to pause, double click to skip to next song etc.
3. It's possible to configure hotkey for it I think, but I haven't tried.

**Comment by Dazaisan99** [comment]
Thanks for the reply ! It works like a charm.

For those wondering how to use your WM like me to control it :

1. Install MPRIS (see https://wiki.archlinux.org/title/MPRIS)
2. Configure your WM accordingly, for example, BSPWM would be

```
alt + {Left, Right, Down}
    playerctl {previous, next, play-pause}
```

Thanks again !

**Comment by tramhao** [comment]
No big deal and thanks for the detail explanation.

**Comment by VuiMuich** [comment]
With `0.7.11` this should be possible now, right?
I was thinking of giving a PR for this a shot, but probably won't have time for it in the next couple weeks.
@tramhao  do you already have plans on implementing this? If not I might look into it after summer.

**Comment by RG-IL** [comment]
I very like this project and i think i could add this quite easily, but I want to see if there is demand for that, should I make a PR for it?

**Comment by hasezoey** [comment]
Yes it is likely to be easily done, but i was still waiting on recommendations on what approach to take, see https://github.com/tramhao/termusic/issues/101#issuecomment-1892556329 (a few comments above).

**Comment by RG-IL** [comment]
I made a PR for it, with the subcommands approach,  you can merge it if you want to
